uint16 task_command_id
uint16 work_type_id
uint16 number_of_items_picked
WorkPresenceArea work_presence_area
CandidateDischargeLocationAreaList candidate_discharge_location